ha_mroonga.hpp

```diff
@@ -694,7 +694,15 @@ class ha_mroonga: public handler
 #endif
   void change_table_ptr(TABLE *table_arg, TABLE_SHARE *share_arg) mrn_override;
   mrn_io_and_cpu_cost scan_time() mrn_override;
+#ifdef MRN_HANDLER_USE_KEYREAD_TIME
+  IO_AND_CPU_COST keyread_time(uint index,
+                               uint ranges,
+                               ha_rows rows,
+                               ulonglong blocks) mrn_override;
+  IO_AND_CPU_COST rnd_pos_time(ha_rows rows) mrn_override;
+#else
   double read_time(uint index, uint ranges, ha_rows rows) mrn_override;
+#endif
```

Comment:
Could you define `MRN_HANDLER_HAVE_READ_TIME` or something and use it instead of `#else`?

```c++
#ifdef MRN_HANDLER_USE_KEYREAD_TIME
#endif
#ifdef MRN_HANDLER_HAVE_READ_KEY
#endif
```

Reply:
I see.
I've fixed it.